### ISSUE TITLE
user interface option for selecting how Compensatory Time should be awarded

### ISSUE THREAD

**Issue by schertza** [City-of-Bloomington/timetrack#121]
With more and more union employees (and non-exempt employess of certain departments), it has been brought up that we need a better way of handling "an employees option" for the awarding of Compensatory Time.

What seems to be "acceptable by all" is when an employee submits their time, if they are "earning" Compensatory Time, they would be given the option to place that Time into an Accrual Bank (comptime earned) or receive it in "Pay", (or a combination of both)  Personally I want to keep wages out of Timetrack so if it is "split" it would be split in units of time not dollars.

One suggestion for the user interface is a "percentage" slider between "Accural Bank" and "pay". Meaning the Compensatory Time balance would be broken down into two pieces defined by the users input. 

Note: if a call-out code is used, that Compensatory time block would always be treated as Pay and would not have the option of going to an Accrual.

All up for discussion but is being driven by "T and D" as well as animal care.

Currently this is being handle by exception with Kelly as the payroll manager. With more and more people joining it would be great to get it back in the hands of the employee.

**Comment by adabutch** [comment]
Let's chat about this new functionality once @charlesbrandt is back, in order to figure out what works here and how we can implement it on both the back & front ends.

**Comment by rbdietz** [comment]
We should consider whether this should possibly be a setting outside of
timekeeping.  This also may be something that isn't essential to have
implemented in 2018 and we should focus time/attention elsewhere.




Rick Dietz

Director, Information & Technology Services Department
City of Bloomington, IN
dietzr@bloomington.in.gov
812.349.3485 Office
812.671.0776 Cell
bloomington.in.gov

**Comment by sibow** [comment]
For union people, we already gave them the option to pick comp-time earned or over-time earned. 
They specify that with hours and fraction of hour such as 1.hr, 1.5 hr etc. This is assuming that they are working more than 8 hours of the day that this was selected on. 
For other special groups, we can do similar but we have to distinguish them from current exempt and non-exempt employees.
 Some T & D employees have a special case where they are considered exempt but they earn their hours after 40 not 45.  Alan and I contacted HR about this and they are studying it and they will contact us later.
